helm/prometheus-operator/README.md

```diff
@@ -80,6 +80,10 @@ Parameter | Description | Default
 `global.hyperkube.repository` | Hyperkube image | `quay.io/coreos/hyperkube`
 `global.hyperkube.tag` | Hyperkube image tag | `v1.6.2_coreos.0`
 `global.hyperkube.pullPolicy` | Hyperkube image pull policy | `IfNotPresent`
+`global.prometheus-config-reloader.repository` | prometheus-config-reloader image | `quay.io/coreos/prometheus-config-reloader`
+`global.prometheus-config-reloader.tag` | prometheus-config-reloader tag | `v0.0.1`
+`global.configmap-reload.repository` | configmap-reload image | `quay.io/coreos/configmap-reload`
+`global.configmap-reload.tag` | configmap-reload tag | `v0.0.1`
 `image.repository` | Image | `quay.io/coreos/prometheus-operator`
```

Comment:
Let's "de-globalize" these. I'll be doing the same for the hyperkube values.

Also, please stick to camelCase.